assets/src/components/post-relationship-control/index.js

```diff
@@ -23,6 +23,7 @@ function PostRelationshipControl( props ) {
 							post_status: postStatus,
 							search: query,
 							filter,
+							additionalParams,
```

Comment:
@i-am-chitti These names are underscored, lets follow this pattern? `additional_params: additionalParams`?